src/migrations/1666213888459-SeasonsUpdating.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class SeasonsUpdating1666213888459 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `Alter TABLE "seasons" ADD COLUMN "year" INTEGER NOT NULL`
+        );
```

Comment:
we should have a migration for the entire season entity, not just for the year column. Did you generate this migration or write it yourself?